packages/config/lib/index.js

```diff
@@ -20,6 +20,7 @@ const breakingKeys = _.map(breakingOptions, 'name')
 const defaultValues = createIndex(options, 'name', 'defaultValue')
 const publicConfigKeys = _(options).reject({ isInternal: true }).map('name').value()
 const validationRules = createIndex(options, 'name', 'validation')
+const readOnlyConfigOptions = options.filter((option) => !option.canUpdateDuringTestTime).map((option) => option.name).reduce((name, val) => ({ ...name, [val]: true }), {})
```

Comment:
Did you look trying to leverage createIndex to create this?
```suggestion
const readOnlyConfigOptions = createIndex(options, 'name', 'canUpdateDuringTestTime')
```